packages/core/onClickOutside/index.ts

```diff
@@ -42,9 +45,13 @@ export function onClickOutside<T extends OnClickOutsideOptions>(
   if (!window)
     return
 
-  let shouldListen = true
+  if (isIOS && !_iOSWorkaround) {
+    _iOSWorkaround = true
+    Array.from(window.document.body.children)
```

Comment:
Are you able to provide some reference to explain why this is needed in the comments?